### ISSUE TITLE
Hebrew dlig does too much

### ISSUE THREAD

**Issue by madig** [googlefonts/googlesans#321]
- https://github.com/googlefonts/googlesans/pull/281#issuecomment-860835541
- https://github.com/googlefonts/googlesans/pull/281#issuecomment-861213125

---

Test string:

```
יו
```

**Comment by chrissimpkins** [comment]
@danielgrumer Will you please provide additional details about what is necessary to fix this issue that you discussed with Nikolaus?  This originated in a separate thread and the links to the original images are in https://github.com/googlefonts/googlesans/issues/321#issue-927243629.  IIUC you feel that we have identified bugs here and I'd like to fix these in our v6 release.

**Comment by danielgrumer** [comment]
Yes sure, In the animation there are 4 lines. 
The dlig feature should result in visual differences only in the 1st one. 
all the other lines should look exactly the same. the 2nd and 4th lines are ok, but the 3rd line changes.

It is probably the order of the components that is incorrect. 

@chrissimpkins Is that clear enough? we can set up a quick meeting if you want me to show you, or that we have a look together and fix this in a minute.

Daniel

**Comment by chrissimpkins** [comment]
Thanks Daniel.

Here is the Yiddish dlig lookup:

```fea
 lookup hb_yiddish {
        lookupflag IgnoreMarks;
        sub alef-hb lamed-hb by aleflamed-hb;
        sub vav-hb vav-hb by vavvav-hb;
        sub yod-hb vav-hb by vavyod-hb;
        sub yod-hb yod-hb by yodyod-hb;
} hb_yiddish;
```

How do you suggest that we change it? Or are you indicating that there is a glyph change required?

**Comment by danielgrumer** [comment]
replace:
`        sub yod-hb vav-hb by vavyod-hb;
`

with:
`        sub vav-hb yod-hb by vavyod-hb;
`

**Comment by chrissimpkins** [comment]
Thanks Daniel!

I opened https://github.com/googlefonts/googlesans/pull/355 with the change that you indicated.

**Comment by chrissimpkins** [comment]
@madig is vav-yod the only dlig issue that you identified in #281?  #355 should close this thread?

**Comment by madig** [comment]
I'm not aware of anything else, except https://github.com/googlefonts/googlesans/pull/281#issuecomment-861637446, which may or may not be a separate thing?

**Comment by chrissimpkins** [comment]
> I'm not aware of anything else, except [#281 (comment)](https://github.com/googlefonts/googlesans/pull/281#issuecomment-861637446), which may or may not be a separate thing?

Thanks Nikolaus!

**Comment by chrissimpkins** [comment]
@danielgrumer Will you please have a look at the files in https://github.com/googlefonts/googlesans/suites/3274336441/artifacts/76280687 to confirm that we fixed the issue with your feature update?  If I understand the issue correctly, it looks like they now shape properly to me.

**Comment by chrissimpkins** [comment]
@danielgrumer I merged the fix so that I can continue with other scheduled work.  Please let us know if this did not address the dlig issue. Thanks!

**Comment by chrissimpkins** [comment]
This fix will be released in v6.000
